docs/04-api-reference/front-commerce-remix/index.mdx

```diff
@@ -232,6 +233,38 @@ between the components. This also works when using the
 [useRevalidator](https://remix.run/docs/en/main/hooks/use-revalidator#userevalidator)
 hook.
 
+### `useApiFetcherPromise`
+
+Just like `useApiFetcher`, `useApiFetcherPromise` allows you to retrieve
+optimised data from `/api` endpoints. As the name suggests, it returns a return
+a promise instead of the actual data to be able to process the data in a more
+"synchronous" way, without depending on fetcher states.
```

Comment:
```suggestion
optimised data from `/api` endpoints. The only difference is that it returns a promise instead of the actual data. Use it when you need to process the data in a more
"synchronous" way, without depending on fetcher states.
```